mobile/envoy_build_config/extension_registry.cc

```diff
@@ -122,7 +122,6 @@ void ExtensionRegistry::registerFactories() {
   Network::forceRegisterGetAddrInfoDnsResolverFactory();
   Network::forceRegisterSocketInterfaceImpl();
   Network::forceRegisterUdpDefaultWriterFactoryFactory();
-  Network::Matching::forceRegisterApplicationProtocolInputFactory();
```

Comment:
Is this one of the extensions that you think I added eagerly to maintain backwards compat when we removed the static extension factory registration and that we don't actually need?

Reply:
yep - we shouldn't need any of the network matchers AFIK.  Working on the rest of them as well.

Reply:
Thank you!